backend/src/repos/memory_repo.py

```
@@ -0,0 +1,76 @@
+from typing import Optional
+
+from langgraph.store.base import BaseStore, SearchItem
+
+from src.repos.base_repo import BaseRepo
+from src.services.db import get_store_in_memory
+from src.schemas.entities.memory import Memory
+from src.schemas.entities import SearchFilter
+
+
+class MemoryRepo(BaseRepo):
+    def __init__(self, user_id: str, store: BaseStore = get_store_in_memory()):
```

Ruff (0.14.14)

[warning] Do not perform function call `get_store_in_memory` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)